src/dos/drive_local.cpp

```diff
@@ -322,7 +334,10 @@ bool localDrive::FindNext(DOS_DTA & dta) {
  	if (~srch_attr & find_attr & (DOS_ATTR_DIRECTORY | DOS_ATTR_HIDDEN | DOS_ATTR_SYSTEM)) goto again;
 	
 	/*file is okay, setup everything to be copied in DTA Block */
-	char find_name[DOS_NAMELENGTH_ASCII];Bit16u find_date,find_time;Bit32u find_size;
```

Comment:
Thank you for cleaning this :) This line was truly horrible - my eyes were moving past it without noticing that there are new variables declared in here.

Reply:
Same here!